docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -4,5 +4,63 @@
 Vitamin A Deficiency
 ====================
 
+Disease Description
+-------------------
+
+Vitamin A deficiency (VAD) is a lack of vitamin A in blood and tissues.
+Vitamin A deficiency is considered as one of the most serious public health concerns in developing countries
+and can contribute directly or indirectly to disability. 
```

Comment:
I'd recommend to add the citation of where this disease description comes from. It could be valuable to add information about the descriptive statistics about VAD, such as how many YLDs or DALYs are associated with this cause.

Reply:
I agree it would be good to add a citation here, and to eventually add more information quantifying the burden of VAD.

Reply:
Will add citation, thank you